features/events/traps/src/test/resources/org/opennms/netmgt/trapd/trapdDefault.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this Spring context. You don't need to mock anything for this test besides a TrapdConfigBean and you don't need any of these objects to do that.